src/materials/DamagePlasticityStressUpdate.C

```diff
@@ -39,6 +39,7 @@ DamagePlasticityStressUpdate::validParams()
       "ft_ep_slope_factor_at_zero_ep",
       "ft_ep_slope_factor_at_zero_ep <= 1 & ft_ep_slope_factor_at_zero_ep >= 0",
       "slope of ft vs plastic strain curve at zero plastic strain");
+  // WGA - unit needed for this?
```

Comment:
Damage is unitless. Stress and strain have units, but I believe we're flexible on what system the user chooses.